core/validation_schema.py

```diff
@@ -371,6 +371,7 @@
             "Programme ID": {"parent_table": "Programme_Ref", "parent_pk": "Programme ID"},
         },
         "non-nullable": ["Programme ID", "Question"],
+        "table_nullable": True,
```

Comment:
I think you may have added this to Round 2 instead of Round 3

Reply:
Which also makes me think, whilst you're here could you re-order the schema's to appear in numerical order in the file? Makes more sense than the random order they're in now

Reply:
haha that's a hard change to check, as the schemas are quite similar, git has only detected partial changes :D It's too clever sometimes...